browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -554,6 +554,21 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_UPDATE_FAILED_CUSTOM_FILTER" desc="Custom filter Update failed.">
         <ph name="TIME">%1$s<ex>1 hours ago</ex></ph> Update failed
       </message>
+      <message name="IDS_CREATE_CUSTOM_FILTERS_TITLE" desc="Title for Create Custom filters.">
+        Create custom filters
+      </message>
+      <message name="IDS_SAVE_CHANGES" desc="Button text for create custom filter lists.">
+        Save changes
+      </message>
+      <message name="IDS_SAVED_CHANGES_SUCCESS" desc="Create custom filter lists success text.">
+        Saved changes successfully
+      </message>
+      <message name="IDS_CREATE_CUSTOM_FILTER_SUMMARY" desc="Create custom filter summary.">
+        Add custom filters you've created, one per line, in the field above. Be sure to use the <ph name="ADBLOCK_FILTER_SYNTAX">%1$s</ph>.
+      </message>
+      <message name="IDS_ADBLOCK_FILTER_SYNTAX" desc="Text for Adblock filter syntax.">
+        Adblock filter syntax
+      </message>
```

Comment:
Is there a reason to separate these into 2 strings instead of using 1 string with <ph name="BEGIN_LINK">...<ph name="END_LINK">?

Reply:
yes, we need string `Adblock filter syntax` separate as it is required here: https://github.com/brave/brave-core/blob/d4123c35e769116d939eea1fbd045036e34ac455/android/java/org/chromium/chrome/browser/shields/CreateCustomFiltersFragment.java#L82C30-L82C51

Reply:
Got it, thanks!